schemas/session-replay/metadata-record-schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "metadata-record-schema.json",
+    "title": "MetadataRecord",
+    "type": "object",
+    "description": "Schema of a Record which contains the screen properties.",
+    "allOf": [
+        {
+            "$ref": "_common-record-schema.json"
+        },
+        {
+            "required": [
+                "type",
+                "data"
+            ],
+            "properties": {
+                "type": {
+                    "type": "integer",
+                    "description": "The type of this Record.",
+                    "const": 4,
+                    "readOnly": true
+                },
+                "data": {
+                    "type": "object",
+                    "description": "The data contained by this record.",
+                    "readOnly": true,
+                    "required": [
+                        "width",
+                        "height"
+                    ],
+                    "properties": {
+                        "width": {
+                            "type": "integer",
+                            "description": "The width of the screen in pixels normalised based on the device pixels per inch density (DPI). Example: if a device has a DPI = 2 the normalized width will be the current width divided by 2.",
+                            "readOnly": true
+                        },
+                        "height": {
+                            "type": "integer",
+                            "description": "The height of the screen in pixels normalised based on the device pixels per inch density (DPI). Example: if a device has a DPI = 2 the normalized height will be the current height divided by 2.",
+                            "readOnly": true
+                        }
```

Comment:
I agree with Thibaut, having the same source field as the rum events would be needed.

Adding it to any record might be a good thing but, in the renderer part we will use the metadata-record to switch into mobile or browser mode.

Reply:
What about the `device_type` ? @ThibautGeriz @mquentin why do you think this one is needed ?

Reply:
> Should this only be in the MetadataRecord ?

yes, this is the initial record that kick everything else in motion, once we know it's all good

> What about the device_type ? why do you think this one is needed ?

It's not needed at the moment but the moment we want to add something related to the platform like keyboard or some phone frame, it's going to be needed, It's better to be thorough I think on the datastructure

Reply:
> Should this only be in the MetadataRecord ?

cf. https://github.com/DataDog/rum-events-format/pull/66#discussion_r913742244
I think it would be useful to have the source for all records